### PR TITLE
[enh] wikipedia infobox

### PR DESCRIPTION
**Pull request by MarcAbonce**
creates simple multilingual infobox using wikipedia's api

extracting the opening paragraph is the only tricky part because the api doesn't provide that directly.

also modified:
- duckduckgo definitions also accpets more languages
- disabled duckduckgo definitions engine (for supported languages it's still better though)
- wikidata: return local link as infobox id (to make it mergeable with wikipedia's)
- results: fixes in merge_two_infoboxes
- template: print bidirectional text

### CODE REVIEW

**Comment by MarcAbonce** [comment]
related to #447 and maybe #446

**Comment by dalf** [comment]
Sorry I wake up late: the duckduckgo definition engine can produce results in other languages, at least French (using the `kl` argument if I remember correctly).

But there is an problem : the [current implementation](https://github.com/asciimoo/searx/blob/master/searx/results.py#L126) uses an `id` which is the [the English wikipedia link](https://github.com/asciimoo/searx/blob/master/searx/engines/duckduckgo_definitions.py#L94) (see also https://github.com/asciimoo/searx/blob/master/searx/engines/wikidata.py#L191 ). So if the duckduckgo definition engine returns no English results, how to merge the infoboxes ?

**Comment by MarcAbonce** [comment]
Actually, it looks like Duckduckgo Definitions takes the language from the Accept-Language http header. The kl argument did nothing to me.
But it still doesn't work for all languages that Wikipedia supports. Chinese, Arabic and Hebrew don't return anything.

To merge the infoboxes, Wikidata now returns the id in the user's language.

**Comment by MarcAbonce** [comment]
Ok, now I also modified duckduckgo_definitions to accept more languages.
The infoboxes merge correctly as far as I've seen.

File: searx/engines/duckduckgo_definitions.py [hunk 1]
```diff
@@ -103,6 +104,10 @@ def response(resp):
         urls.append({'title': search_res.get('DefinitionSource'),
                     'url': definitionURL})
 
+    # to merge with wikidata's infobox
+    if infobox_id:
+        infobox_id = sub(r'^http:', r'https:', infobox_id)
```
Comment (asciimoo) [inline searx/engines/duckduckgo_definitions.py]: Compiled regex would be more efficient

File: searx/engines/duckduckgo_definitions.py [hunk 2]
```diff
@@ -103,6 +106,10 @@ def response(resp):
         urls.append({'title': search_res.get('DefinitionSource'),
                     'url': definitionURL})
 
+    # to merge with wikidata's infobox
+    if infobox_id:
+        infobox_id = sub(http_regex, 'https:', infobox_id)
```
Comment (asciimoo) [inline searx/engines/duckduckgo_definitions.py]: thanks the fix,
this could be also `http_regex.sub('https:', infobox_id)`

**Comment by asciimoo** [comment]
great, thanks

**Pull request merged**